controllers/machinehealthcheck_targets.go

```diff
@@ -111,6 +118,7 @@ func (t *healthCheckTarget) needsRemediation(logger logr.Logger, timeoutForMachi
 			return false, timeoutForMachineToHaveNode
 		}
 		if t.Machine.Status.LastUpdated.Add(timeoutForMachineToHaveNode).Before(now) {
+			conditions.MarkFalse(t.Machine, clusterv1.ConditionHealthCheckSucceeded, "NodeTimeoutReached", clusterv1.ConditionSeverityWarning, "Duration: %s", timeoutForMachineToHaveNode.String())
```

Comment:
```suggestion
			conditions.MarkFalse(t.Machine, clusterv1.ConditionHealthCheckSucceeded, "NodeStartupTimeout", clusterv1.ConditionSeverityWarning, "Node failed to report startup in %s", timeoutForMachineToHaveNode.String())
```
(plus, use a constant for condition reason)